Packs/GitHub/Integrations/GitHub/GitHub.yml

```
@@ -117,6 +117,9 @@ script:
     - contextPath: GitHub.Issue.Closed_by
       description: User who closed the issue.
       type: String
+    - contextPath: GitHub.Issue.Organization
+      description: The repository owner.
```

update in all places (yml and readme)
```suggestion
      description: The repository organization.
```